docs/repo-docs/guides/tools/typescript.mdx

```diff
@@ -73,7 +73,9 @@ Inside `packages/typescript-config`, you have a few `json` files which represent
 }
 ```
 
-<LinkToDocumentation href="https://www.typescriptlang.org/tsconfig">`tsconfig` options reference</LinkToDocumentation>
+<LinkToDocumentation href="https://www.typescriptlang.org/tsconfig">
+  `tsconfig` options reference
```

Comment:
This newline will mess up the rendered output by adding an extra <p> tag.

Reply:
ugg. autoformatter here.  big oof. fixed in 3ea0cf313aac82ff12e6e728942121617efdf7da

Reply:
Yeah, no worries. I only know because it bit me the other day. 😅